tests/fixtures/on_demand_cluster_fixtures.py

```diff
@@ -119,6 +120,24 @@ def ondemand_k8s_cluster(request):
     return cluster
 
 
+@pytest.fixture(scope="session")
+def ondemand_k8s_docker_cluster(request):
+    kube_config_path = Path.home() / ".kube" / "config"
+
+    if not kube_config_path.exists():
+        pytest.skip("no kubeconfig found")
+
+    args = {
+        "name": "k8s-docker-cpu",
+        "provider": "kubernetes",
+        "instance_type": "CPU:1",
+        "memory": ".2",
+        "image_id": "docker:ubuntu:18.04",
```

Comment:
Maybe we should do a more specific one (e.g. a Ray or SkyPilot one) to make the testing have to accomodate existing Python / package versions already installed?